xml/System.Net.Mail/SmtpClient.xml

```diff
@@ -35,7 +35,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Allows applications to send email by using the Simple Mail Transfer Protocol (SMTP).</summary>
+    <summary>Allows applications to send email by using the Simple Mail Transfer Protocol (SMTP). The <c>SmtpClient</c> type is now obsolete.</summary>
```

Comment:
Is there are recommended alternative?  That's going to be an obvious question for readers when they see this.

Reply:
The recommended alternatives are listed in the obsolete message, which appears in the syntax section, @BillWagner. There is no .NET base class library alternative.